vars/managerPipeline.groovy

```diff
@@ -54,6 +54,10 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('backup_bucket_backend', '')}",
                description: 's3|gcs|azure or empty',
                name: 'backup_bucket_backend')
+            string(defaultValue: "${pipelineParams.get('backup_bucket_location', '')}",
```

Comment:
worth to consider for refactor the concept of `backup_bucket_location` which is used as `backup bucket name`and sometimes is used as a list in the code (by split). I'm really confused what it really is.

Reply:
Ah, I see this param is used in some nemesis and functional tests (`test_functional`).
So, refactoring it across all these tests and rerunning them will take time.
I'd suggest to go without this fix for now as this PR doesn't introduce this complication but just uses already existing approach. We can do refactoring in a separate ticket.
@soyacz What do you think?

Reply:
yes, can go to separate PR.